ggml/src/ggml-backend-reg.cpp

```diff
@@ -534,8 +534,13 @@ static ggml_backend_reg_t ggml_backend_load_best(const char * name, bool silent,
     fs::path best_path;
 
     for (const auto & search_path : search_paths) {
-        if (!fs::exists(search_path)) {
-            GGML_LOG_DEBUG("%s: search path %s does not exist\n", __func__, path_str(search_path).c_str());
+        std::error_code ec;
+        if (!fs::exists(search_path, ec)) {
```

Comment:
Can be further simplified using if-init
```suggestion
        std::error_code ec;
        if (std::error_code ec; !fs::exists(search_path, ec)) {
```